src/renderer/components/WatchVideoDescription/WatchVideoDescription.vue

```diff
@@ -86,6 +80,16 @@ if (props.descriptionHtml !== '') {
   }
 }
 
+const processedShownDescription = computed(() => {
+  if (shownDescription === '') { return shownDescription }
+
+  return processDescriptionHtml(shownDescription, linkTabIndex.value)
+})
+
+const linkTabIndex = computed(() => {
+  return showFullDescription.value ? 0 : -1
```

Comment:
**Non-blocking**: As the values are only used inside of strings, it makes sense to write them as string directly to avoid them needing to be converted from a number into a string at runtime.
```suggestion
  return showFullDescription.value ? '0' : '-1'
```